plugin/rooter.vim

```diff
@@ -155,7 +155,7 @@ endfunction
 
 " Mappings and commands {{{
 
-if !hasmapto("<Plug>RooterChangeToRootDirectory")
+if !exists("g:rooter_disable_mapping") && !hasmapto("<Plug>RooterChangeToRootDirectory")
```

Comment:
This should be `g:rooter_disable_map` for consistency with the other occurences.